models/address/fields.yaml

```
@@ -1,8 +1,12 @@
 fields:
     customer:
-        label: 'offline.mall::lang.order.customer'
+        label: Customer
```

```suggestion
        label: 'offline.mall::lang.order.customer'
```

Let's use the original label here.